src/components/button-bar/button-bar.style.ts

```diff
@@ -28,6 +28,7 @@ const StyledButtonBar = styled.div<StyledButtonBarProps>`
     button {
       margin: 0;
       border: 2px solid var(--colorsActionMajor500);
+      border-radius: 0px;
```

Comment:
You'll need to do more than just remove the radius on the buttons. It will require using css selectors to target first and last to apply the radius etc

```
button {
  ...
  border-radius: var(--borderRadius000);
  ...
  
  :first-of-type {
     border-top-left-radius: var(--borderRadius400);
     border-bottom-left-radius: var(--borderRadius400);
  }
  
  :last-of-type {
     border-top-right-radius: var(--borderRadius400);
     border-bottom-right-radius: var(--borderRadius400);
   }
}
```
Achieves:

![image](https://user-images.githubusercontent.com/44157880/216068347-08e25aab-01ff-4ab1-b1a1-ae95905fd9a6.png)